ale/base/data_naif.py

```diff
@@ -685,44 +723,19 @@ def spiceql_mission(self):
 
     def spiceql_call(self, function_name = "", function_args = {}):
         """
-        Interface to SpiceQL (Spice Query Library) for both Offline and Online use
-
-        This function will access the value passed through props defined as `web`. This
-        value determines the access pattern for spice data. When set to Online, you will
-        access the SpiceQL service provided through the USGS Astro AWS platform. This service
-        performs kernel and data aquisition. If set to Offline, you will access locally loaded
-        kernels, and SpiceQL will do no searching for you.
-
-        Parameters
-        ----------
-        functions_name : str
-                         String defineing the function to call, properly exposed SpiceQL
-                         functions should map 1-to-1 with endpoints on the service
-                        
-        function_args : dict
-                        Dictionary of arguments used by the function
-
-        Returns : any
-                  Any return from a SpiceQL function
-        """
-        if not self.use_web:
-            func = getattr(pyspiceql, function_name)
-
-            # Ensure that in offline mode we anticipate the user loading/passing their own kernels
-            # to ALE
-            function_args["searchKernels"] = self.use_web
-            return func(**function_args)
-
-        try:
-            url = "http://localhost:9000/2015-03-31/functions/function/invocations"
-            headers = {
-                'Content-Type': 'application/x-www-form-urlencoded',
-            }
-            function_args["func"] = function_name
-            r = requests.get(url, data=json.dumps(function_args), headers=headers, verify=False)
-            r.raise_for_status()
-            if r.json()["statusCode"] != 200:
-                raise requests.HTTPError(f"Recieved code {r.json()['statusCode']} from spice server, with error: {r.json()}")
-            return r.json()["body"]["return"]
-        except requests.exceptions.HTTPError as err:
-            raise err
+        Driver based wrapper for accessing spice data through spiceql
+
+        See Also
+        --------
+        ale.kernel_access.spiceql_call
+
+        Returns
+        -------
+        : json
+          Json data from the SpiceQL call
+        """
+        # Ideally this would work if a user passed no kernels but still
+        # set ISISDATA
+        if (not hasattr(self, "kernels")):
+            function_args["searchKernels"] = True
+        return kernel_access.spiceql_call(function_name, function_args, self.use_web)
```

Comment:
JSON is not a python return type. Is this a string that happens to be JSON formatted or is it a dict? The caller will want to know that and then can parse to JSON or not as appropriate.

Reply:
It's a dict, brain was havin a hard time